cdap-ui/app/cdap/components/PipelineList/DeployedPipelineView/DeployedPipelineView.scss

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+$header-height: 70px;
+$padding-size: 45px;
+
+.pipeline-deployed-view {
+  padding-left: $padding-size;
+  padding-right: $padding-size;
+
+  .deployed-header {
+    height: $header-height;
+    line-height: $header-height;
+
+    .pipeline-count,
```

Comment:
Can we have the styling for `pipeline-count` to be under the component. Would be easier when we clean up/refactor things.